cli/filplus.go

```diff
@@ -334,14 +400,81 @@ var filplusListClaimsCmd = &cli.Command{
 		defer closer()
 		ctx := ReqContext(cctx)
 
-		providerAddr, err := address.NewFromString(cctx.Args().Get(0))
-		if err != nil {
-			return err
-		}
+		writeOut := func(tsHeight abi.ChainEpoch, claims map[verifreg.ClaimId]verifreg.Claim, json, expired bool) error {
+			// Map Keys. Corresponds to the standard tablewriter output
+			allocationID := "ClaimID"
+			provider := "Provider"
+			client := "Client"
+			data := "Data"
+			size := "Size"
+			tMin := "TermMin"
+			tMax := "TermMax"
+			tStart := "TermStart"
+			sector := "Sector"
+
+			// One-to-one mapping between tablewriter keys and JSON keys
+			tableKeysToJsonKeys := map[string]string{
+				allocationID: strings.ToLower(allocationID),
+				provider:     strings.ToLower(provider),
+				client:       strings.ToLower(client),
+				data:         strings.ToLower(data),
+				size:         strings.ToLower(size),
+				tMin:         strings.ToLower(tMin),
+				tMax:         strings.ToLower(tMax),
+				tStart:       strings.ToLower(tStart),
+				sector:       strings.ToLower(sector),
+			}
 
-		providerIdAddr, err := api.StateLookupID(ctx, providerAddr, types.EmptyTSK)
-		if err != nil {
-			return err
+			var allocs []map[string]interface{}
+
+			for key, val := range claims {
+				if tsHeight > val.TermMax || !expired {
+					alloc := map[string]interface{}{
+						allocationID: key,
+						provider:     val.Provider,
+						client:       val.Client,
+						data:         val.Data,
+						size:         val.Size,
+						tMin:         val.TermMin,
+						tMax:         val.TermMax,
+						tStart:       val.TermStart,
+						sector:       val.Sector,
+					}
+					allocs = append(allocs, alloc)
+				}
+			}
+
+			if json {
+				// get a new list of wallets with json keys instead of tablewriter keys
+				var jsonAllocs []map[string]interface{}
+				for _, alloc := range allocs {
+					jsonAlloc := make(map[string]interface{})
+					for k, v := range alloc {
+						jsonAlloc[tableKeysToJsonKeys[k]] = v
+					}
+					jsonAllocs = append(jsonAllocs, jsonAlloc)
+				}
+				// then return this!
+				return PrintJson(jsonAllocs)
```

Comment:
```suggestion
				// get a new list of claims with json keys instead of tablewriter keys
				var jsonClaims []map[string]interface{}
				for _, claim := range claims {
					jsonClaim := make(map[string]interface{})
					for k, v := range claim {
						jsonAlloc[tableKeysToJsonKeys[k]] = v
					}
					jsonClaims = append(jsonClaims, jsonClaim)
				}
				// then return this!
				return PrintJson(jsonClaims)
```